readme.md

```diff
@@ -75,14 +75,13 @@ There are two ways to run Renovate on Azure DevOps:
 
 Go to the Visual Studio Marketplace and install the [Renovate Me](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me) extension in your organization.
 From there you can create a pipeline with the `RenovateMe` task.
-[More details on how to configure the pipeline](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me).
```

Comment:
This is the same link as above (line 76, `Renovate me`), this extra link to the same site just confused me.